modules/installation-azure-stack-hub-config-yaml.adoc

```diff
@@ -81,3 +90,85 @@ endif::openshift-origin[]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
+endif::ash[]
+
+ifdef::ash-default[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+credentialsMode: Manual
+controlPlane:
+  name: master
+  replicas: 3
+compute:
+- name: worker
+  platform: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1> <2>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+ifndef::openshift-origin[]
+  networkType: OpenShiftSDN
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+  networkType: OVNKubernetes
+endif::openshift-origin[]
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  azure:
+    armEndpoint: azurestack_arm_endpoint <1> <3>
+    baseDomainResourceGroupName: resource_group <1> <4>
+    region: azure_stack_local_region <1> <5>
+    resourceGroupName: existing_resource_group <1> <6>
+    outboundType: Loadbalancer
+    cloudName: AzureStackCloud <1>
+    clusterOSimage: https://vhdsa.blob.example.example.com/vhd/rhcos-410.84.202112040202-0-azurestack.x86_64.vhd <1> <7>
```

Comment:
@patrickdillon As you mentioned, users must upload a storage blob that contains an RHCOS VHD. Is this the section of the UPI doc that you said covers this? https://docs.openshift.com/container-platform/4.9/installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.html#installation-azure-user-infra-uploading-rhcos_installing-azure-stack-hub-user-infra

If so, the prereqs state that you have generated the Ignition config files. Given that this is IPI, is this prereq still required or would this be a step that the installer handles for the user?

Reply:
That is the correct section. As we discussed in the call, the prereqs about ignition configs do not apply. In fact the image must be uploaded before the the install config.

The relevant steps are 1-7. 

Note that the UPI steps imply that the image go into the same resource group as the cluster. The VHD can go into any storage blob as long as the blob is public and in the same environment.

Reply:
Added Uploading the RHCOS cluster image procedure before the cluster installation steps.

Reply:
> While you mentioned that steps 1-7 are relevant, is the first command of step 8 also required?

No. That is for the ignition, which is not needed in IPI.